lms/djangoapps/ccx/views.py

```diff
@@ -301,6 +303,14 @@ def create_ccx(request, course, ccx=None):
             user=request.user,
         )
 
+    # .. event_implemented_name: COURSE_CREATED
+    COURSE_CREATED.send_event(
+        time=datetime.now(timezone.utc),
+        course=CourseData(
+            course_key=CCXLocator.from_course_locator(course.id, str(ccx.id)),
```

Comment:
There is already a variable with the  ccx locator. @JuanDavidBuitrago 
```suggestion
            course_key=ccx_id,
```